README.md

```diff
@@ -1 +1,99 @@
-# lookout-sdk
\ No newline at end of file
+lookout SDK [![GoDoc](https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg)](https://godoc.org/github.com/src-d/lookout-sdk) [![PyPI version](https://badge.fury.io/py/lookout-sdk.svg)](https://pypi.org/project/lookout-sdk/) [![Build Status](https://travis-ci.org/src-d/src-d/lookout-sdk.svg)](https://travis-ci.org/src-d/src-d/lookout-sdk)
+-----------
+
+What is lookout SDK?
+===================
+Lookout SDK is toolkit for writing new analyzers for the [lookout](https://github.com/src-d/lookout/).
```

Comment:
Lookout SDK is **a** toolkit

> new analyzers for the lookout
That sound weird to me. I'd use "the lookout project", or simply "for lookout".